### PR TITLE
bulk operations on User Detail comments

### PR DESCRIPTION
**Pull request by riley**
## What does this PR do?

adds the ability to change the status of comments in bulk from the User Detail panel

## How do I test this PR?

- go to the moderation queue and click the username on a comment to open the UserDetail panel
- tick a few checkboxes
- the UI should show the # of selected comments
- Accept/Reject the comments.

### CODE REVIEW

**Review by kgardnr — APPROVED** [review]
This LGTM once design approves :100:

File: client/coral-admin/src/routes/Moderation/components/UserDetail.js
Comment (alinhle) [inline client/coral-admin/src/routes/Moderation/components/UserDetail.js]: It would be great to bind these callbacks only once, instead on every render, to allow future optimizations.

File: client/coral-admin/src/routes/Moderation/components/UserDetail.js
Comment (alinhle) [inline client/coral-admin/src/routes/Moderation/components/UserDetail.js]: See above

File: client/coral-admin/src/routes/Moderation/components/UserDetail.js
Comment (alinhle) [inline client/coral-admin/src/routes/Moderation/components/UserDetail.js]: See above

File: client/coral-admin/src/routes/Moderation/components/UserDetail.js
Comment (alinhle) [inline client/coral-admin/src/routes/Moderation/components/UserDetail.js]: See above

File: client/coral-admin/src/routes/Moderation/components/Comment.js
Comment (alinhle) [inline client/coral-admin/src/routes/Moderation/components/Comment.js:82]: It would be great to have this defined once, instead on every render. This would be more performant and also allow future optimisations like only rendering when necessary.

**Review by alinhle — CHANGES_REQUESTED** [review]
Small changes for better performance.

**Review by alinhle — APPROVED** [review]
Approving the code. Maybe someone wants to test the new deselect feature, and that the `rejected` tab contains the newly rejected comments before merging.

**Pull request merged**
